src/tests/UnitTests/IO/LogEntryOutputProcessorTests.cs

```diff
@@ -61,6 +61,49 @@ public void ShouldParseSingleCommit()
             AssertProcessOutput(output, expected);
         }
 
+        [Test]
+        public void ShouldParseSummaryAndDescription()
+        {
+            var output = new[]
+            {
+                "1cd4b9154a88bc8c7b09cb8cacc79bf1d5bde8cf",
+                "865b8d9d6e5e3bd6d7a4dc9c9f3588192314942c",
+                "Author Person",
+                "author@example.com",
+                "2017-01-06T15:36:57+01:00",
+                "Author Person",
+                "author@example.com",
+                "2017-01-06T15:36:57+01:00",
+                "Rename RepositoryModelBase to RepositoryModel",
+                "",
+                "This is a line on the description",
+                "---GHUBODYEND---",
+                "M       src/GitHub.App/Models/RemoteRepositoryModel.cs",
+                null,
+            };
+
+            var commitTime = new DateTimeOffset(2017, 1, 6, 15, 36, 57, TimeSpan.FromHours(1));
+
+            var expected = new[]
+            {
+                new GitLogEntry("1cd4b9154a88bc8c7b09cb8cacc79bf1d5bde8cf",
+                    "Author Person", "author@example.com",
+                    "Author Person", "author@example.com",
+                    "Rename RepositoryModelBase to RepositoryModel",
+                    Environment.NewLine + "This is a line on the description",
```

Comment:
Wouldn't we want to remove the newline from the Body?

Reply:
I'm not sure if the GitLogEntry should be trim whitespace when parsing things, but maybe it should, yeah

Reply:
Did this change?